ui/src/main/resources/Meeting/Code/Migrations/MigrationMacros.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+-->
+
+<xwikidoc version="1.2" reference="Meeting.Code.Migrations.MigrationMacros" locale="">
+  <web>Meeting.Code.Migrations</web>
+  <name>MigrationMacros</name>
+  <language/>
+  <defaultLanguage/>
+  <translation>0</translation>
+  <creator>xwiki:XWiki.Admin</creator>
+  <creationDate>1556173254000</creationDate>
+  <parent>Meeting.WebHome</parent>
+  <author>xwiki:XWiki.Admin</author>
+  <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
+  <date>1556174841000</date>
+  <contentUpdateDate>1556174803000</contentUpdateDate>
+  <version>1.1</version>
+  <title>MigrationMacros</title>
+  <comment/>
+  <minorEdit>false</minorEdit>
+  <syntaxId>xwiki/2.1</syntaxId>
+  <hidden>false</hidden>
+  <content>{{velocity}}
+#macro (testMigrationNeeded)
+  #set($query = "from doc.object(Meeting.Code.MeetingClass) as meeting")
+  #set($meetings = $services.query.xwql($query).execute())
+  #set($migrationNeeded = false)
+  #set($migration191 = false)
+  #set($migration195 = false)
+  #foreach($meet in $meetings)
+    #set($meetDoc = $xwiki.getDocument($meet))
+    #set($meetObj = $meetDoc.getObject('Meeting.Code.MeetingClass'))
+    #set($durationString = $meetObj.getValue('durationString'))
+    #set($meetingTitle = $meetObj.getValue('meetingTitle'))
```

Comment:
You should be consistent in naming variables. `meet` or `meeting`, just use the same.